packages/x-components/src/x-modules/popular-searches/components/popular-searches.vue

```diff
@@ -102,13 +132,25 @@ You can use your custom implementation of the Popular Search's content. In the e
 instead of using the default Popular Search's content, an icon is added, as well as a span with the
 query of the Popular Search's suggestion.
 
-```vue
-<PopularSearches>
-  <template #suggestion-content="{ suggestion }">
-    <img class="x-popular-search__icon" src="./popular-search-icon.svg" />
-    <span class="x-popular-search__query">{{ suggestion.query }}</span>
-  </template>
-</PopularSearches>
+```vue live
+<template>
+  <PopularSearches>
+    <template #suggestion-content="{ suggestion }">
+      <img class="x-popular-search__icon" src="/assets/icons/bulb.svg" />
```

Comment:
Change this icon for the bulb and import it.

Reply:
I am trying to add more specific icons, matching this component, i added `TrendingIcon`.